manifests/charts/istio-operator/values.yaml

```diff
@@ -1,8 +1,6 @@
 hub: gcr.io/istio-testing
 tag: latest
 
-operatorNamespace: istio-operator
```

Comment:
@ostromart @morvencao  is this a backwards incompatible change? What will happen to users setting this?

Reply:
Ping @ostromart @morvencao

Reply:
@ostromart  @morvencao Any thoughts on this ?

Reply:
up @ostromart  @morvencao

Reply:
up @ostromart @morvencao